ui/src/app/shared/components/webcomponents/confirmation-modal.component.ts

```diff
@@ -72,4 +121,33 @@ export class ConfirmationModalComponent {
     this.confirmText.valueChanges.subscribe(() => this.isValid = this.confirmText.value.toLocaleLowerCase() == 'delete');
 
   }
+  ngOnInit(){
+  }
+
+
+
+  openLinkedEntity(linkedEntity) {
+    let entityUrl;
+    if(this.modalData?.entityUrl)
+      entityUrl = this.formatEntityUrlIfExists(this.modalData?.entityUrl, linkedEntity);
+    if(!entityUrl) {
+      entityUrl = "/ui/td/cases/" + linkedEntity?.id + "/steps";
```

Comment:
plz avoid hard coded string, move things to enum

Reply:
done, check latest commit